docs/kits/Business Partner Kit/Documentation BPDM/page_api-reference.md

```diff
@@ -0,0 +1,45 @@
+---
+id: API Reference
+title: Api Reference
+description: ''
+sidebar_position: 2
+---
+
+The BPDM is comprised of two primary services: BPDM Pool and BPDM Gate. The link provided below are for a Swagger UI interface, which is an interactive tool for exploring and testing RESTful APIs. Swagger UI provides a user-friendly interface for interacting with APIs, allowing you to make requests, view responses, and explore different API endpoints.
+
+To navigate through the Swagger UI interface, you can use the sidebar navigation menu on the left-hand side of the screen. This menu provides a hierarchical view of the available API endpoints and operations, allowing you to drill down into specific endpoints and view the associated request parameters, response schemas, and sample responses.
+
+## BPDM Gate Api reference
+
+A gate for a member to share business partner data with CatenaX.
+
+Swagger: [https://partners-gate.int.demo.catena-x.net/ui/swagger-ui/index.html](https://partners-gate.int.demo.catena-x.net/ui/swagger-ui/index.html)
+
+As you can see it, Gate service consist of four major controllers which are listed below.
+
+1. Site Controller
+2. legal Entity Controller
+3. Address Controller
+4. Business Partner Controller
```

Comment:
Listing the controllers that are already listed in the Swagger/Open-API doesn't help much. Better add an explanation to what these Controllers are responsible for.